src/sparql/ordo-add-annotation-based-mappings1.ru

```diff
@@ -0,0 +1,29 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix IAO: <http://purl.obolibrary.org/obo/IAO_>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix def: <http://purl.obolibrary.org/obo/IAO_0000115>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix obo: <http://purl.obolibrary.org/obo/>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+
+INSERT {
+  ?cls a owl:Class;
+     	skos:narrowMatch ?value .
+} WHERE {
```

Comment:
We also want to delete the old pattern.

Reply:
@matentzn Ok, I can do that. However, are you sure we want to do that? Wouldn't this be lossy, since the ORDO mapping strings contain more information than the relationship CURIEs we're adding/replacing them with.

Reply:
Done. Now deletes that pattern.

However, some things I want to bring up at next meeting. I will point out specifically in some new review comments on my new code.